opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-csv-domain.ts

```diff
@@ -21,6 +21,12 @@ import { extractContentFrom } from '../../utils/fileToContent';
 import { isCompatibleVersionWithMinimal } from '../../utils/version';
 import { FunctionalError } from '../../config/errors';
 import { convertRepresentationsIds } from '../internal/mapper-utils';
+import { addUser } from '../../domain/user';
+import { findAll } from '../../domain/group';
```

Comment:
```suggestion
import { findAll as findAllGroup } from '../../domain/group';
```
When it's a find or findAll from another domain, please give it a alias to have better code readability.